crates/symbolicator/src/services/symbolication.rs

```diff
@@ -2103,36 +2205,39 @@ impl SymbolicationActor {
             tracing::debug!("Processing minidump ({} bytes)", len);
             metric!(time_raw("minidump.upload.size") = len);
 
-            let client_stacktraces = ByteView::open(&minidump_file)
-                .map_or(Ok(None), |bv| parse_stacktraces_from_minidump(&bv));
-
             let mut cfi_caches = CfiCacheModules::new();
 
-            let result = self
-                .stackwalk_minidump_iteratively_with_breakpad(
-                    scope.clone(),
-                    minidump_file,
-                    sources.clone(),
-                    &mut cfi_caches,
-                )
-                .await?;
+            let future = self.stackwalk_minidump_iteratively(
+                scope.clone(),
+                &minidump_file,
+                sources.clone(),
+                &mut cfi_caches,
+                options.compare_stackwalking_methods,
+            );
 
+            let (result_old, new_stackwalking_problem) = match future.await {
+                Ok(result) => result,
+                Err(err) => {
+                    self.maybe_persist_minidump(minidump_file);
+                    return Err(err);
+                }
+            };
+
+            metric!(timer("minidump.stackwalk.duration") = result_old.duration, "method" => "old");
```

Comment:
this feels a bit odd out here, the other metric is emitted somewhere very different.